ThunderCats/UITextField+TCAdditions.m

```diff
@@ -40,4 +40,27 @@ - (void)tc_setLeftAndRightPadding:(CGFloat)paddingWidth
     self.rightViewMode = UITextFieldViewModeUnlessEditing;
 }
 
+- (void)tc_addToolbarWithNextButton:(UITextField * __tc_nonnull)nextTextField
```

Comment:
The signature on this method is a little strange.  `withNextButton:` implies that I should pass a `UIButton` that serves as the next button.  I'm not sure of a better name right now, maybe @lgauthier has some input here, but I'm not sure I like this name.